exercises/simple-cipher/src/test/java/IncorrectKeyCipherTest.java

```diff
@@ -0,0 +1,54 @@
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Step 3: Incorrect key cipher
+ */
+public class IncorrectKeyCipherTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Before
+    public void setup() {
+    }
+
+    /**
+     * Throws an error with an all uppercase key
+     */
```

Comment:
I don't think you need these comments. You should be able to use the test name to convey what it's testing 🙂